set.mm

```diff
@@ -629570,8 +629561,8 @@ standardize vectors to a length (norm) of one, but such definitions require
     $( Version of ~ hbsb2 with a disjoint variable condition, which does not
        require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
     hbsb2w $p |- ( -. A. x x = y -> ( [ y / x ] ph -> A. x [ y / x ] ph ) ) $=
-      ( weq wal wn wsb wi sb4bw sb2w axc4i syl6bi ) BCDZBEFABCGZMAHZBENBEABCION
-      BABCJKL $.
+      ( weq wal wn wsb sb4bw alim stdpc4 syl6com biimprd pm2.61i axc4i syl6bi
```

Comment:
The idea of BJ was to replace sb2w with one direction of sb6 (i.e. sb6 followed by biimpri).  Is there a technical reason for using alim stdpc4 syl6.com biimprd, pm2.61i instead?

Reply:
I forgot, thanks.

Reply:
> EDIT: sb4bw is also a weakened form of sb6, that should be fully replaced, if this will not hamper the minimizing process.

I'm going to check out this one as well.

Reply:
Removed sb4bw.

Reply:
Yes, @wlammen is right: this is what I meant in general. In this particular case, we can actually do without hbsb2w altogether and use hbs1